Directory.Build.props

```
@@ -8,7 +8,7 @@
 		<ApplicationIcon>../resources/ASF.ico</ApplicationIcon>
 		<Authors>JustArchi</Authors>
 		<Company>JustArchiNET</Company>
-		<Copyright>Copyright © 2015-2021 JustArchiNET</Copyright>
+		<Copyright>Copyright © 2015-$([System.DateTime]::UtcNow.ToString("yyyy")) JustArchiNET</Copyright>
```

```suggestion
		<Copyright>Copyright © 2015-$([System.DateTime]::UtcNow.Year) JustArchiNET</Copyright>
```

Not tested, give it a try, if it doesn't work add `.ToString()`, much faster.

---

Yes that works as well 😄